scripts/run-ci.sh

```diff
@@ -29,6 +29,9 @@ if [ ! -z "$CPPLINT" ]; then
     echo "Running cpplint ..."
     find -name "*.cc" -o -name "*.h" | sort | xargs ./scripts/cpplint.py
     ./scripts/check-emscripten-enums.sh
+
+    echo "Running gofmt ..."
+    ./scripts/check-gofmt.sh || (echo "Go files are not properly formatted." ; exit 1)
```

Comment:
Running `./scripts/check-gofmt.sh` should be sufficient. This script is `set -e`, so it will fail if any of the called scripts fail (which `check-gofmt.sh` will do on errors).

Reply:
But it won't show a nice error message :)
Anyway, removed the `echo` call.